src/pyOpenMS/tests/test_ProForma.py

```
@@ -589,5 +620,447 @@ def test_peptidoform_len():
 
+# =============================================================================
+# Spectrum Generation Tests
+# =============================================================================
+
+def test_can_generate_spectrum_simple():
+    """Test canGenerateSpectrum for simple peptides."""
+    import pyopenms as p
+
+    # Simple peptide should be generatable
+    pf = p.Peptidoform.fromString("PEPTIDE")
+    assert pf.canGenerateSpectrum() == True
```

Ruff (0.14.13)

Avoid equality comparisons to `True`; use `pf.canGenerateSpectrum():` for truth checks

Replace with `pf.canGenerateSpectrum()`

(E712)